### DISCUSSION TITLE
Good Repo and some queries/suggestions

### DISCUSSION THREAD

**Discussion by skwskwskwskw** [deepdoctection/deepdoctection#104] [General]
First of all, thanks for the great repo. 

Some queries: 
1. May I know how's the model being trained? 
2. How to download the weights and model files?
3. Possible to change config? 
4. Also, I think having an option to set pytesseract path would be helpful.

**Comment by JaMe76** [comment]
Thank you for your feedback. 

1.) Layout, cell and row/column detection have been trained with **Tensorpack** framework (as it is the first one I started using). You can find the training scripts for cell detection [here](https://deepdoctection.readthedocs.io/en/latest/tutorials/scripts/). All models for document layout analysis have been trained in the same way. Trained Tensorflow models have been converted into Detectron2 models using [this script](https://github.com/deepdoctection/deepdoctection/blob/master/scripts/tp2d2.py).

2.) [This notebook](https://github.com/deepdoctection/notebooks/blob/main/Custom_Pipeline.ipynb) shows you how to do this. Check the section **Catalog and registries**. 

3.) This is difficult to answer without having a something specific in mind. You can change some settings for inference but not those that will change the model structure. You can change settings regarding the model structure when training a model from scratch. 

4.) Pytesseract is not being used, only Tesseract. With respect to your suggestion, please check PR #102

**Comment by skwskwskwskw** [comment]
**(4) Figured out about Tesseract issue.** 

Do put the "tessdata" in the system variable as such (for Windows user):

![image](https://user-images.githubusercontent.com/7208012/209800965-daef92ee-674b-4f0a-b1e6-1843496ca815.png)

**Comment by JaMe76** [comment]
With the latest PR you can now change path via

```
import deepdoctection as dd

dd.set_tesseract_path("path/to/tesseract")
```

Unfortunately, I cannot provide advise to Windows users.

**Reply by skwskwskwskw** [reply]
Noted - but I do have other questions/queries on another discussion thread.
